src/lua/compat.h

```diff
@@ -14,7 +14,17 @@ extern "C" {
 
 struct lua_State;
 
+/* toggler of mp_char2escape */
+int
+msgpuck_esc_slash_toggle(struct lua_State *L);
+
+/* toggler for json.encode behavior change */
+int
+json_esc_slash_toggle(struct lua_State *L);
+
 static const struct luaL_Reg compat_internals[] = {
+	{"msgpuck_esc_slash_toggle", msgpuck_esc_slash_toggle},
+	{"json_esc_slash_toggle", json_esc_slash_toggle},
```

Comment:
It is defined in the serializer module, so it is better to expose it from the serializer module. See the review comment for the definition of the function regarding its name. Don't forget to adjust the comment to tarantool's style.

Reply:
> It is defined in the serializer module, so it is better to expose it from the serializer module.

I've tried that. The built-in registration doesn't work as intended (or isn't initialized when required).